classes/class-wc-connect-nux.php

```diff
@@ -261,12 +232,9 @@ public static function get_banner_type_to_display( $status = array() ) {
 				This is an existing user. Do nothing.
 			*/
 			switch ( $status['jetpack_connection_status'] ) {
-				case self::JETPACK_NOT_INSTALLED:
-				case self::JETPACK_INSTALLED_NOT_ACTIVATED:
-				case self::JETPACK_ACTIVATED_NOT_CONNECTED:
+				case self::JETPACK_NOT_CONNECTED:
 					return 'before_jetpack_connection';
 				case self::JETPACK_CONNECTED:
-				case self::JETPACK_DEV:
```

Comment:
I don't think we should present either the `after_jetpack_connection` or the `tos_only_banner` on local or dev sites (i.e. in offline mode) so I am removing this case.